### PR TITLE
Pdn 413.newguideline.components.batch2

### PR DESCRIPTION
**Pull request by jeremie-lucca**
New guidelines components update

### CODE REVIEW

File: packages/scss/src/components/inputs/switch/_switch.scss [hunk 1]
```diff
@@ -16,38 +15,44 @@
...
 
 .switch-label {
-	color: _theme("commons.disabled.color");
 	cursor: pointer;
 	display: inline-block;
-	line-height: _component("switch.label.line-height");
-	padding: _component("switch.label.padding");
-	transition: color 150ms ease;
+	line-height: 1.5rem;
+	padding: 0 0 0 calc(2rem - 4px + #{_theme("spacings.smaller")});
```
Comment (lucienbertin) [inline packages/scss/src/components/inputs/switch/_switch.scss]:
```suggestion
	padding: 0 0 0 calc(2rem - 4px + #{_theme("spacings.smaller", true)});
```
you need to fetch the real value here before `calc` not point towards the css v ar

File: packages/scss/src/components/inputs/switch/_switch.scss [hunk 2]
```diff
@@ -16,38 +15,44 @@
...
 		display: block;
-		height: _component("switch.input.handler.size");
-		left: calc(-1 * ( #{_component("switch.input.handler.size")} - #{_component("switch.input.size")} ) / 2);
+		height: calc(1rem - 4px);
+		left: 2px;
 		position: absolute;
-		top: calc(( #{_component("switch.input.handler.size")} - #{_component("switch.input.size")} ) / 2);
-		transition: all 150ms ease;
-		width: _component("switch.input.handler.size");
+		top: calc(#{_theme("spacings.smallest")} + 2px);
```
Comment (lucienbertin) [inline packages/scss/src/components/inputs/switch/_switch.scss]:
```suggestion
		top: calc(#{_theme("spacings.smallest", true)} + 2px);
```
same

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-940

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-940

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-940

**Pull request merged**